src/match.c

```diff
@@ -1973,6 +1991,32 @@ char* CompilateDemoName()
 	{
 		strlcat(demoname, va("ra_%d", (int)CountPlayers()), sizeof(demoname));
 	}
+	else if (isCA())
+	{
+		if (cvar("k_clan_arena") == 1)
+		{
+			strlcat(demoname, "ca", sizeof(demoname));
+		}
+		else
+		{
+			strlcat(demoname, "wipeout", sizeof(demoname));
+		}
+
+		char teams[MAX_CLIENTS][MAX_TEAM_NAME];
```

Comment:
Could you move this to the start of the block, please? I hate to see variable declarations in the middle :).
Is this even compiling? This is not ISO-C, isn't it?

Reply:
Fixed it. I'm a total C newb so I wouldn't know :) It was compiling though!

Reply:
;)